account_banking_reconciliation/views/account_banking_reconciliation.xml

```diff
@@ -68,32 +68,46 @@
-                                    <field name="currency_id" readonly="1"/>
+                                    <field name="date" readonly="1"
+                                           force_save="1"/>
+                                    <field name="name" readonly="1"
+                                           force_save="1"/>
+                                    <field name="ref" readonly="1"
+                                           force_save="1"/>
+                                    <field name="partner_id" readonly="1"
+                                           force_save="1"/>
+                                    <field name="amount" readonly="1"
+                                           force_save="1"/>
+                                    <field name="amountcur" readonly="1"
+                                           force_save="1"/>
+                                    <field name="currency_id" readonly="1"
+                                           force_save="1"/>
                                     <field name="research_required"
-                                           readonly="1"/>
+                                           readonly="1" force_save="1"/>
                                 </form>
                                 <tree string="Deposits, Credits, and Interest"
                                       editable="top"
                                       mute_additem="draft,to_be_reviewed,done, cancel">
                                     <field name="cleared_bank_account"/>
-                                    <field name="date" readonly="1"/>
-                                    <field name="name" readonly="1"/>
-                                    <field name="ref" readonly="1"/>
-                                    <field name="partner_id" readonly="1"/>
-                                    <field name="amount" readonly="1"/>
-                                    <field name="amountcur" readonly="1"/>
-                                    <field name="currency_id" readonly="1"/>
+                                    <field name="date" readonly="1"
+                                           force_save="1"/>
+                                    <field name="name" readonly="1"
+                                           force_save="1"/>
+                                    <field name="ref" readonly="1"
+                                           force_save="1"/>
+                                    <field name="partner_id" readonly="1"
+                                           force_save="1"/>
+                                    <field name="amount" readonly="1"
+                                           force_save="1"/>
+                                    <field name="amountcur" readonly="1"
+                                           force_save="1"/>
+                                    <field name="currency_id" readonly="1"
+                                           force_save="1"/>
```

Comment:
Just asking:
Do we have all these fields coming from `onchange` ? else `force_save` is not necessary 
(Seems bad Database schema if this is the case here)